src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -88,7 +88,8 @@ internal LibraryViewController(Window dynamoView, ICommandExecutive commandExecu
 
         private void Browser_ZoomFactorChanged(object sender, EventArgs e)
         {
-            dynamoViewModel.Model.PreferenceSettings.LibraryZoomScale = ((float)browser.ZoomFactor);
+            //Multiplies by 100 so the value can be saved as a percentage
+            dynamoViewModel.Model.PreferenceSettings.LibraryZoomScale = (float)browser.ZoomFactor * 100;
```

Comment:
If we use this approach, the number should be int instead of float right?

Reply:
Yes, changed the variable to int